### PR TITLE
[EDCO-14] BannerLink Component

### PR DESCRIPTION
**Pull request by mmoio**
## Description

Adds [BannerLink Component](https://www.figma.com/file/6Hn7Wxmvg6eCSqsMrIP7VU/CMS---UI-Components-%26-Templates?node-id=1%3A16084&t=HTODP8OLfiDqwOEH-1) & related stories

Fixes # (edco-14)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- [x] Storybook
- [x] Visual testing

### CODE REVIEW

File: src/components/BannerLink/BannerLink.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { alpha, Box, Container, Stack, useTheme } from '@mui/material';
+import { Ctas, type CtaProps } from '../../components/Ctas';
+import { type CommonProps } from 'types/components';
+import { type BannerLinkContentProps, Content } from './Content';
+import LoginImg from '../../assets/svg/Login';
```
Comment (stratoivandiluccio) [inline src/components/BannerLink/BannerLink.tsx:5]: I think we should abstract the icon to shown and not use a specific one, like you are doing here.

File: src/assets/images/Login.svg
Comment (vito80ba) [inline src/assets/images/Login.svg:1]: Assets for stories must be in the "/stories/assets" folder

File: src/assets/svg/Login.tsx
Comment (vito80ba) [inline src/assets/svg/Login.tsx:1]: Assets for stories must be in the "/stories/assets" folder

**Review by vito80ba — CHANGES_REQUESTED** [review]

File: src/assets/svg/Login.tsx
Comment (mmoio) [reply src/assets/svg/Login.tsx:1]: done

File: src/components/BannerLink/BannerLink.tsx (hunk 1 above)
Comment (mmoio) [reply src/components/BannerLink/BannerLink.tsx]: Ok, now the component accepts a `decoration` prop of type `Generic` to be rendered above the title.

**Review by vito80ba — APPROVED** [review]

**Review by vito80ba — APPROVED** [review]

**Review by stratoivandiluccio — APPROVED** [review]

**Pull request merged**